SOURCES/etc/xapi.d/plugins/xcpngutils/__init__.py

```diff
@@ -123,6 +134,8 @@ def wrapper(*args, **kwds):
         except EnvironmentError as e:
             message = e.strerror if e.strerror is not None else str(e.args)
             raise XenAPIPlugin.Failure(str(e.errno), [message, str(e.filename), traceback.format_exc()])
+        except ProcessException as e:
+            raise_plugin_error(e.returncode, str(e), backtrace=traceback.format_exc())
```

Comment:
`raise_plugin_error` accepts a "details" parameter that we leave empty here. Maybe we could put the contents of `stdout` (and, possibly, stderr) in it, to prevent `message` from being too big?

Reply:
That's the suggestion, yes.